Makefile

```diff
@@ -66,6 +66,26 @@ build-storybook: install generate-api-types  ## Builds the storybook.
 
 # MISC
 
+.PHONY: scraper
+scraper: ## Inserts the scrapers scraped data into the database.
+	cd scraper && npm install && mkdir -p data && npm run start:full
```

Comment:
better call it scraper-start-full or similar. I would also love to have a scraper-insert to just insert (as the full process takes very long).